gpkit/tests/t_sub.py

```diff
@@ -169,16 +173,16 @@ def setup(self):
             concatm = a | b
             concatm.cost = a.cost*b.cost
             concat_cost = concatm.solve(verbosity=0)["cost"]
-            self.assertAlmostEqual(concat_cost, 0.0109361)
+            self.assertAlmostEqual(concat_cost, 0.0109361)  # 1 cm/1 yd
         a1, b1 = Above(), Below()
         m = a1 & b1
         m.cost = m["x"]
         sol = m.solve(verbosity=0)
         if not isinstance(m["x"].key.units, str):
-            self.assertAlmostEqual(sol["cost"], 0.032808399)  # 1 cm/1 yd
+            self.assertAlmostEqual(sol["cost"], 0.032808399)  # 1 cm/1 ft
         self.assertIn(m["x"], sol["variables"])
-        self.assertIn(a1["x"], sol["variables"])
-        self.assertIn(b1["x"], sol["variables"])
+        #self.assertIn(a1["x"], sol["variables"])
+        #self.assertIn(b1["x"], sol["variables"])
```

Comment:
There's a design discussion to have here, but short answer is yes. Confusing answer is yes, but after LinkingConstraints are figured out.